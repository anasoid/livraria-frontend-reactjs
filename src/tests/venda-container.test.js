import React from 'react';
import renderer from 'react-test-renderer';
import {shallow} from 'enzyme';
import VendaContainer from '../components/venda/venda-container';


describe('test VendaContainer component', () => {

  const wrapper = shallow(<VendaContainer/>);

  it('should renders without crashing ', () => {
    shallow(<VendaContainer/>);
    const tree = renderer.create(<VendaContainer/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should be "Vendas" h1 title ', () => {
    expect(wrapper.find('h1')).toHaveText('Vendas');
  });

});

