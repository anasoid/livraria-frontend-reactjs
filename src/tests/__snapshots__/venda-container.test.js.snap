// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test VendaContainer component should renders without crashing  1`] = `
<div>
  <h1>
    Vendas
  </h1>
  <div>
    Loading Chart
  </div>
</div>
`;
